test/test.js

```diff
@@ -14,7 +14,13 @@ function createRepo( callback ) {
 
 		testRepo.exec( "commit", "--allow-empty",
 			"--author=John Doe <john.doe@example.com>", "-m first commit",
-			callback );
+			function( error ) {
+				if ( error ) {
+					throw error;
```

Comment:
So that we're not calling 'git tag' if something went wrong with the commit?

It's what you did before the commit so I figured it was the right thing to do.